api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -97,6 +97,20 @@ type HTTP1Settings struct {
 	// By default, Envoy will lowercase all the headers.
 	// +optional
 	PreserveHeaderCase *bool `json:"preserveHeaderCase,omitempty"`
+	// HTTP10 turns on support for HTTP/1.0 and HTTP/0.9 requests.
+	// +optional
+	HTTP10 *HTTP10Settings `json:"http10,omitempty"`
+}
+
+// HTTP10Settings provides HTTP/1.0 configuration on the listener.
+type HTTP10Settings struct {
+	// SetHostHeader defines if the HTTP/1.0 request is is missing the Host header,
+	// then the hostname associated with the listener should be injected into the
+	// request.
+	// If this is not set and an HTTP/1.0 request arrives without a host, then
+	// it will be rejected.
+	// +optional
+	SetHostHeder *bool `json:"setHostHeader,omitempty"`
```

Comment:
I know I proposed `set` earlier, would `addHostHeader` be a better name ?

Reply:
Since the host header is only added if it's missing, how about `defaultHost`?  That's also closer to what the Envoy Proxy configuration key is called.

Reply:
@envoyproxy/gateway-maintainers any preferences here ?

Reply:
+1 for `useDefaultHost`

Reply:
lets go with `useDefaultHost` unless @zhaohuabing disagrees :)

Reply:
If `addHostHeaderWhenMissing` is too long, then `useDefaultHost`. :-)